jsx/Form.js

```diff
@@ -370,7 +370,7 @@ class SearchableDropdown extends Component {
     let optionList = [];
     if (sortByValue) {
       for (let key in options) {
-        if (options.hasOwnProperty(key)) {
```

Comment:
Can't these loops just be changed to `for .. of ..` and the hasOwnProperty check be removed?

Reply:
The for..of loop only supports iterable objects like arrays, not objects. I usually use a combination of for of and Object.entries.

Reply:
That still sounds cleaner than Object.prototype.hasOwnProperty.call

Reply:
Hi @driusan & @laemtl, please see the latest commit for the change.

Reply:
The latest commit looks like it only updated 1 place, the rest of the lines in the PR are still using "hasOwnProperty"

Reply:
Last one taking longer to fix because I want to test it and I'm needing to configure couchdb.

Reply:
@driusan @laemtl okay fixed